README.md

```diff
@@ -1,164 +1,123 @@
+release and download the binary that is provided.
 
-## To play
-
-### Install Rust
-If you'd like to play with Polkadot, you'll need to install a client like this
-one. First, get Rust (1.39.0 or later) and the support software if you don't already have it:
+### Install via Cargo
 
+If you want to install Polkadot in your PATH, you can do so with with:
 
 ```bash
-curl https://sh.rustup.rs -sSf | sh
+cargo install --force --git https://github.com/paritytech/polkadot --tag <version> polkadot
 ```
 
-You may need to add Cargo's bin directoy to your PATH environment variable. Restarting your computer will do this for you automatically. Once done, finish installing the support software:
+### Build from Source
+
+If you'd like to build from source, first install Rust. You may need to add Cargo's bin directory
+to your PATH environment variable. Restarting your computer will do this for you automatically.
 
 ```bash
-sudo apt install make clang pkg-config libssl-dev
+curl https://sh.rustup.rs -sSf | sh
 ```
 
 If you already have Rust installed, make sure you're using the latest version by running:
 
-
 ```bash
 rustup update
 ```
 
-### Install "Kusama CC-3" Canary Network
-
-Build Kusama by cloning this repository and running the following commands from the root directory of the repo:
-
-```bash
-git checkout master
-./scripts/init.sh
-cargo build --release
-```
-
-Connect to the global Kusama canary network by default by running:
+Once done, finish installing the support software:
 
 ```bash
-./target/release/polkadot --name "hello world!"
+sudo apt install make clang pkg-config libssl-dev
 ```
 
-You can see your node on [telemetry].
-
-[telemetry]: https://telemetry.polkadot.io/#list/Kusama%20CC3
-
-### Install PoC-4 on "Alexander" Testnet
-
-Build Polkadot PoC-4 by cloning this repository and running the following commands from the root directory of the repo:
+Build the client by cloning this repository and running the following commands from the root
+directory of the repo:
 
 ```bash
-git checkout v0.4
+git checkout <latest tagged release>
 ./scripts/init.sh
-./scripts/build.sh
 cargo build --release
 ```
 
-If you were previously running PoC-3 on this testnet, you may need to purge your chain data first:
+## Networks
 
-```bash
-./target/release/polkadot purge-chain
-```
+This repo supports runtimes for Polkadot, Kusama, and Westend.
 
-Finally, connect to the global "Alexander" testnet by default by running:
+### Connect to Polkadot Chain Candidate 1 (CC1)
 
-```bash
-./target/release/polkadot
-```
-
-### Install PoC-2 "Krumme Lanke" Testnet
-
-Install Polkadot PoC-2 and have a `polkadot` binary installed to your `PATH` with:
-
-```
-cargo install --git https://github.com/paritytech/substrate.git --branch v0.2 polkadot
-```
-
-Connect to the global "Krumme Lanke" testnet by default by running:
+Connect to the global Polkadot CC1 network by running:
 
 ```bash
-polkadot
+./target/release/polkadot --name "hello world!" --chain=polkadot
```

Comment:
maybe replace "hello world!" with something unique or leave it without explicit name?